Rakefile

```diff
@@ -18,6 +18,7 @@ end
 
 PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp", "vendor/**/*.pp"]
 PuppetLint.configuration.log_format = '%{path}:%{line}:%{KIND}: %{message}'
+PuppetLint.configuration.send('disable_arrow_on_right_operand_line')
```

Comment:
This is modulesynced so will be overwritten. I'll see if I can come up with patches to just fix the code to match the new style.